src/kms/entry.rs

```diff
@@ -46,7 +46,7 @@ pub struct KeyEntry {
     /// Thumbprints for the key
     pub(crate) thumbprints: Vec<String>,
     /// Thumbprints for the key
-    pub(crate) tags: Vec<EntryTag>,
+    pub tags: Vec<EntryTag>,
```

Comment:
Note the tags are kept sorted to facilitate equality checks, and an outside caller might not maintain that. Can the KeyEntry just expose a reference to the tags?

Reply:
Done